api/src/app.ts

```diff
@@ -52,7 +52,7 @@ async function getTauriVersion(): Promise<string> {
  * @returns {Promise<void>} Application is closing, nothing is returned
  */
 async function exit(exitCode: Number = 0): Promise<void> {
```

Comment:
```suggestion
async function exit(exitCode: number = 0): Promise<void> {
```
`number` is the actual typescript type.

Reply:
@amrbashir Feel free to fix it yourself if you find anything else, I just needed to patch what prevented me to generate the docs.